### PR TITLE
Small cleaning fixes for integration tests

### PR DESCRIPTION
**Pull request by Marketionist**
**This pull request updates integration tests**

Also:
- added trailing comma in ```test_settings.py```
- removed one level of nesting in page creation test

### CODE REVIEW

**Comment by FinalAngel** [comment]
LGTM

**Comment by FinalAngel** [comment]
@Marketionist tests are failing here

**Comment by Marketionist** [comment]
Yes, I'm on it now

File: aldryn_newsblog/tests/frontend/integration/specs/spec.newsblog.crud.js [hunk 1]
```diff
@@ -81,33 +81,32 @@ describe('Aldryn Newsblog tests: ', function () {
...
-                            }, newsBlogPage.mainElementsWaitTime);
-
-                            // validate test link text
-                            newsBlogPage.testLink.getText()
-                                .then(function (title) {
-                                expect(title).toEqual('Test');
-                            });
-                        }
-                    });
+                    newsBlogPage.slugErrorNotification.isPresent();
```
Comment (vxsx) [inline aldryn_newsblog/tests/frontend/integration/specs/spec.newsblog.crud.js]: return this promise

**Pull request merged**